### DISCUSSION TITLE
Node.js

### DISCUSSION THREAD

**Discussion by Oyebolade** [rupali-codes/LinksHub#183] [Q&A]
How can I view the site on my local computer after cloning it?

**Comment by AzouKr** [comment]
To view a site on your local computer after cloning it:

1. Navigate to inside the project folder on terminal, where I would hopefully see a package.json file
2. You have to Install the necessary dependencies by running `npm install`.
3. Do an `npm install` for installing all the project dependencies
4. Do an `npm install -g nodemon` for installing all the project dependencies
5. Then `npm start` OR `node app.js` OR `nodemon app.js` to get the app running on local host (if your .js file is not named app you use your file name).
6. use nodemon app.js ( nodemon is a utility that will monitor for any changes in your source and automatically restart your server)

**Comment by Oyebolade** [comment]
Thanks,
What's the difference between number 3 & 4?

**Reply by AzouKr** [reply]
`npm install` is a command to install dependencies listed in the package.json file of a Node.js project. It installs packages locally in the node_modules folder of the current project directory.

`npm install -g nodemon` is a command to install the nodemon package globally (-g means globally ) on your system, which means that it will be available to all Node.js projects on your computer. 

`nodemon `is a tool that automatically restarts Node.js applications when file changes are detected, making it a popular tool for development.

**Reply by Oyebolade** [reply]
All right,
Thanks
